core/src/test/java/org/elasticsearch/action/bulk/byscroll/DeleteByQueryRequestTests.java

```diff
@@ -74,4 +75,23 @@ protected void extraRandomizationForSlice(DeleteByQueryRequest original) {
     protected void extraForSliceAssertions(DeleteByQueryRequest original, DeleteByQueryRequest forSliced) {
         // No extra assertions needed
     }
+
+    @Test
```

Comment:
the Test annotation is banned in our project :) Something should fail when running the build from gradle. if the test method starts with test we are good, no need for the annotation